.azure-pipelines/azure-pipelines.yml

```diff
@@ -7,6 +7,9 @@ trigger:
   branches:
     include:
       - master
+  tags:
+    include:
+      - '*'
```

Comment:
why are you adding this now?

Reply:
I relealised we were missing it when it didn't run on the previous tagged build